transport/server.cc

```diff
@@ -698,12 +698,21 @@ future<> cql_server::connection::process_request() {
             });
             auto istream = buf.get_istream();
             (void)_process_request_stage(this, istream, op, stream, seastar::ref(_client_state), tracing_requested, mem_permit)
-                    .then_wrapped([this, buf = std::move(buf), mem_permit, leave = std::move(leave)] (future<foreign_ptr<std::unique_ptr<cql_server::response>>> response_f) mutable {
+                    .then_wrapped([this, buf = std::move(buf), mem_permit, leave = std::move(leave), stream] (future<foreign_ptr<std::unique_ptr<cql_server::response>>> response_f) mutable {
```

Comment:
Yes, handling big CQL batch ended with error, we tried to write this error to the log and failed since it was longer that 65k. The client got timeout and the cause of the error was lost.

OK, I'll create an issue and try to write an explicit test for this.